src/renderer/components/swap/Swap.tsx

```diff
@@ -271,6 +271,8 @@ export const Swap = ({
           // Decimal needs to be converted back for using orginal decimal of source asset
           amount: convertBaseAmountDecimal(amountToSwapMax1e8, sourceAssetDecimal),
           memo: getSwapMemo({ asset: target, address, limit: swapResultAmountMax1e8.times(1.0 - slipTolerance * 0.01) })
+            .slice(0, -3)
+            .concat(ASGARDEX_SWAP_IDENTIFIER)
```

Comment:
Pls extract handling of  `getSwapMemo` into a helper of `Swap.uitls` and add a test for it.